kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/thirdChallenge/MainThirdChallenge.java

```diff
@@ -1,20 +1,16 @@
 package com.kodilla.good.patterns.challenges.thirdChallenge;
 
-import java.util.Map;
-
 public class MainThirdChallenge {
 
     public static void main(String[] args) {
 
-        Map<Integer, Flight> mapOfFlights = new FlightDB().initMap();
-
-        new FlightFinder(mapOfFlights).printAirportsInUse(mapOfFlights);
-        new FlightFinder(mapOfFlights).scanFlightsToAirportWithAnotherAirport("Szymany",
+        new FlightFinder().printAirportsInUse();
```

Comment:
Skoro każdy taki sam ten FlightFinder, to po co go za każdym tazem tworzyć?

Moim zdaniem w klasie main powinieneś stworzyć mapę lotów i przekazać ją w konstruktorze do FlighFinder.
Uprość nazwy metod, tak jak rozmawialiśmy w środę:
    findFrom(String from)
    findTo(String to)
    findFromToVia(String from, String via, String to)
i te metody będą korzystały z mapy w klasie FlightFiner'a jako pola obiektu.